### PR TITLE
Minor modification on TiFlash 5.0 docs

### PR DESCRIPTION
**Pull request by ilovesoup**
### What is changed, added or deleted? (Required)
Polish limitation section.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
https://github.com/pingcap/docs/pull/5177

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@ilovesoup Could you please invite a technical review?

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -277,14 +277,20 @@ mysql> explain select count(*) from customer c join nation n on c.c_nationkey=n.
...
-> - 在配置项 `new_collations_enabled_on_first_bootstrap`(/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) 的值为 `true` 时，MPP 不支持 join 的连接键类型为字符串或 `group by` 聚合运算时列类型为字符串的情况。在处理这两类查询时，默认不选择 MPP 模式。
-
 TiFlash 提供了两个全局/会话变量决定是否选择 Broadcast Hash Join，分别为：
 
 - [`tidb_broadcast_join_threshold_size`](/system-variables.md#tidb_broadcast_join_threshold_count-从-v50-ga-版本开始引入 )，单位为 bytes。如果表大小（字节数）小于该值，则选择 Broadcast Hash Join 算法。否则选择 Shuffled Hash Join 算法。
 - [`tidb_broadcast_join_threshold_count`](/system-variables.md#tidb_broadcast_join_threshold_count-从-v50-ga-版本开始引入 )，单位为行数。如果 join 的对象为子查询，优化器无法估计子查询结果集大小，在这种情况下通过结果集行数判断。如果子查询的行数估计值小于该变量，则选择 Broadcast Hash Join 算法。否则选择 Shuffled Hash Join 算法。
+
+> **注意：**
+>
+> TiFlash 计算层不支持：
```
Comment (tisonkun) [inline tiflash/use-tiflash.md:287]: The English version is smooth for "doesn't support: - ... - ..." but the Chinese version isn't.

I'd prefer either

```
TiFlash 计算层不支持：

- 数值溢出检测。...
- Window Function
- 从 TiKV 读取数据

除此以外，MPP 模式还不支持：

- 分区表。...
- ...
```

or consider a smooth Chinese version, use the sentence pattern "TiFlash 有以下局限性：- 不支持 ...", "MPP 模式额外有以下局限性： - 不支持分区表 ..." and (maybe) change the English version correspondingly.

**Comment by zanmato1984** [comment]
This PR's content has been merged with PR https://github.com/pingcap/docs-cn/pull/5891. Please move forward to the latter and close this one.

**Pull request closed without merging**